src/test/java/uk/gov/hmcts/reform/hmc/validator/CapitalizedEnumPatternValidatorTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.hmc.validator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.hmc.model.DayOfWeekUnAvailableType;
+import uk.gov.hmcts.reform.hmc.model.UnavailabilityDow;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
The use of `MockitoExtension.class` here looks redundant

Reply:
Fixed in https://github.com/hmcts/hmc-cft-hearing-service/pull/132/commits/d183666a5e408a22c50e2e84c81733c9bf910103